.gitignore

```diff
@@ -1,3 +1,4 @@
 ﻿.vs
 bin
 obj
+/LunrCore.sln.DotSettings.*
```

Comment:
You can be less specific and exclude `*.sln.DotSettings.*`

Reply:
Fixed